sparkdl/horovod/__init__.py

```diff
@@ -13,3 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# pylint: disable=wrong-import-position
+# pylint: disable=invalid-name
```

Comment:
Why do we need these given there is no import or name here?

Reply:
It is minor one, we can leave it to reduce conflicts.